assets/src/components/SurveyModal.js

```diff
@@ -41,6 +41,10 @@ const useStyles = makeStyles((theme) => ({
     right: 0,
     width: '100%',
     height: '100%'
+  },
+  surveyButton: {
+    color: theme.palette.primary.main,
+    background: theme.palette.getContrastText(theme.palette.primary.main),
```

Comment:
This feels kinda backwards (getting the `contrastText` color to use as a background?), but it works, probably not worth worrying about. I guess the alternative would be just using `white` from Material UI (`import { white } from '@material-ui/core/colors'`) or the hex value, but when I tried it seemed to affect the hover styling. I think what you have is fine.

Reply:
Using `white` might make the color match the current background more, but it also might make the button less noticeable...

Reply:
Yeah, seemed to me too but it works the way we expected.

I didn't want to import white because was somewhat concerned of what it would look like if someone had the colors different than ours, like if they had a lighter or white color for the top bar. I was hoping for some calculation that would work with custom theming, though I didn't try it with any alternative themes.

Reply:
So if I set
`    "PRIMARY_UI_COLOR" : "#ffffff"`

Then the box comes up black which seems like what we'd want here.
![image](https://user-images.githubusercontent.com/27447/107074596-f231f080-67b6-11eb-8eaa-f34ba8cc7b7c.png)

Reply:
Oh, you are correct about using `white`. I don't think anyone would use a really light color in the app bar, but it's better to handle that case.

Reply:
Yeah, I'd first tried it with a light yellow and it was still a darker color and looked pretty good with the light yellow color. Not sure what color combos places might have.